eo-maven-plugin/src/main/java/org/eolang/maven/ResolveMojo.java

```diff
@@ -95,6 +98,14 @@ public final class ResolveMojo extends SafeMojo {
     @SuppressWarnings("PMD.ImmutableField")
     private BiConsumer<Dependency, Path> central;
 
+    /**
+     * Check transitive dependencies.
+     *
+     * @checkstyle MemberNameCheck (7 lines)
+     */
+    @SuppressWarnings({"PMD.ImmutableField", "PMD.AvoidFieldNameMatchingMethodName"})
```

Comment:
@volodya-lombrozo this is a valid concern of PMD: AvoidFieldNameMatchingMethodName. Don't use similar names for methods and attrs

Reply:
@yegor256 fixed.